parsing/src/main/resources/Parser.jj

```diff
@@ -6589,9 +6590,13 @@ SqlNode NamedFunctionCall() :
             call = SqlStdOperatorTable.OVER.createCall(s.end(over), call, over);
         }
     ]
-    {
-        return call;
-    }
+    (
+<#list parser.postExpressionMethods as method>
```

Comment:
nit: Since this feature is for dialect 1 only, are we assuming that any parser (in the future) which has postExpressionMethods means that they would support this feature? I think we are implying whether or not to add this behavior by the existence of postExpressionMethods in a parser. I am not as familiar to this feature as you do, so could you please give me more context information on this?

Reply:
Sure, this is similar to how we were templating in lists of methods that only belong in babel. We would put the array, such as postExpressionMethods, in every config.fmpp file but only fill it with method names in the babel config.fmpp. Now we're just filling the array in the Dialect1 config file. If the array is empty in other dialect config files, nothing will be printed by the <#list> tag for that dialect parser.

Reply:
Thank you for explaining it in details. What I am concern about is that if someone want to use one of the postExpressionMethods in the future in some other parser, when they add a method to the config.fmpp of that parser, this parsing behavior(not requiring parentheses for CAST/FORMAT and NAMED) will be added (unintentionally?) too.

Reply:
I think I see what you mean. Perhaps making a separate array for postNamedFunctionCallMethods could solve this?

Reply:
Hmmm, I think adding another parser config boolean variable and add it to the condition statement when deciding if this feature is needed or not will help. For example, nested the <#list> </#list> statement with an <#if> statement. I don't know if I am suggesting the best way though, please try out other solution as you see fit.

Reply:
Yeah I think what I pushed now with an if statement works well. If some methods need to be added later for only the named function call, then another list could be added for that